index.html

```diff
@@ -0,0 +1,173 @@
+                2019 Design Journal among other peer reviewed journals.
+            </p>
+        </li>
+        <li>
+            <img src="https://cdn.glitch.com/74171c12-683c-4072-8113-bc7e1b6a35cc%2FSuhailaSikand%20copy.jpg?v=1607029785493"
+                alt="smiling close-up of a dark haired woman with outdoor plants behind her" />
+            <a class="speaker-name" href="http://www.suhaila.me/" target="blank">Suhaila Sikand</a>
+            <p>Climate designer, exploring community, equity, behavior, & design</p>
+            <p>2PM-2:45PM</p>
+            <p>
+                As a social impact and climate designer, I am grounded in community, equity, and sustainability.
+                Unfortunately, human-centered design doesn’t always support these principles with the right intentions.
+                Instead of seeing a problem in a community, let’s get invited to work with the community. Instead of
+                interpreting people’s needs, let’s include them in the process. <strong>Human centered design implies a
+                    surface
+                    level engagement where the power of design remains with the designer, not the community.</strong>
+            </p>
+        </li>
+    </ul>
+    <section id="register">
+        <h2>- register -</h2>
+    </section>
+
+    <form>
+        <ul>
+            <li>
+                <label for="name">name:</label>
+                <input type="text" name="user_name">
+            </li>
+            <li>
+                <label for="mail">email:</label>
+                <input type="email" name="user_email">
+            </li>
+            <li>
+                <label for="phone-number">phone:</label>
+                <input type="phone" name="phone" </li> <li>
+                <label for="msg">question(s):</label>
+                <textarea name="user_message"></textarea>
+            </li>
+            <li class="button">
+                <button class="button-reset" type="submit">send</button>
+            </li>
+        </ul>
+    </form>
+
+</body>
+
+
+<footer>
+    <nav>
+        <ul>
+            <li>
+                <a href="#details">Details</a>
+            </li>
+            <li>
+                <a href="#speakers">Speakers</a>
+            </li>
+            <li>
+                <a href="#register">Register</a>
+            </li>
+            <li>
+                <a href="#">Previous events</a>
+            </li>
+            <li>
+                <a href="#">Careers</a>
+            </li>
+            <li>
+                <a href="#">Contact</a>
+            </li>
+            <li>
+                <a href="#">Privacy policy</a>
+            </li>
+            <li>
+                <a href="#">Code of conduct</a>
+            </li>
+            <li>
+                <label for="mail">Sign up for newsletter:</label>
+                <input type="email" name="user_email">
```

Comment:
This needs its own `form` element and `button` to submit the form, like so:

```suggestion
<form>
                <label for="mail">Sign up for newsletter:</label>
                <input id="mail" type="email" name="user_email">
                <button type="submit">Button Text Here</button>
</form>
```